app/src/components/Navigation/Navbar/Navbar.tsx

```diff
@@ -24,7 +24,9 @@ export const Navbar = ({
           title="Log out"
           variant="outline"
         />
-        <Button color="light" onClick={onClick} title={navButton} />
+        {navButton && (
+          <Button color="light" onClick={onClick} title={navButton} />
+        )}
```

Comment:
if you bass the button component in to the navButton prop you wont have to pass the style and the onclick as props as well. e.g.
```
<Navbar navButton={<Button title="..." onClick={...} color="...}
```